build.proj

```diff
@@ -34,6 +34,8 @@
       The provided path should be ended to a `\` character without white spaces: Ex. C:\x86\
     -->
     <DotnetPath></DotnetPath>
+    <!-- Using these properties to compile and pack netfx dll fixes nuget package explorer error "Compiler flags: missing" -->
```

Comment:
It doesn't make sense to me why we should repeat these properties here. Are you sure there isn't a better solution?

Reply:
Non-SDK base projects such as .Net Framework needs the properties to be supplied in for nuget pack command to work properly. However, since this only affects NetFx projects, the NugetPackProperty definition was moved to NetFX project file instead.